snap/local/etc/zookeeper/log4j.properties

```diff
@@ -46,7 +46,7 @@ log4j.appender.CONSOLE.layout.ConversionPattern=%d{ISO8601} [myid:%X{myid}] - %-
 #
 log4j.appender.ROLLINGFILE=org.apache.log4j.RollingFileAppender
 log4j.appender.ROLLINGFILE.Threshold=${zookeeper.log.threshold}
-log4j.appender.ROLLINGFILE.File=${zookeeper.log.dir}/${zookeeper.log.file}
+log4j.appender.ROLLINGFILE.File=${zookeeper.log.dir}/zookeeper.log
```

Comment:
**question** isn't this the same given, that in line 25 we set the key `zookeeper.log.file` ? If the key is not working I'd probably also remove that from there

Reply:
This is the bug. We set `zookeeper.log.file` here, but the startup script for ZK overrides it - https://github.com/apache/zookeeper/blob/e50a0bbae1bc0b812ac2656f5ba3f9eab7c3d62e/bin/zkServer.sh#L152-L153

Reply:
Aweful. Truly aweful to override envs like this :angry: